### PR TITLE
Makman2/regfix

### PR DESCRIPTION
**Pull request by Makman2**

### CODE REVIEW

**Comment by Nosferatul** [comment]
ack 98878eb

**Comment by Nosferatul** [comment]
ack 633a164
ack bc47dac

**Comment by Makman2** [comment]
ack 633a164

**Comment by Mixih** [comment]
thought there was discussion about commenting but maybe not....

**Comment by Mixih** [comment]
ack b2ec316 4b31525

**Comment by Mixih** [comment]
ack cc769c5

**Comment by Mixih** [comment]
ack db70ce1

**Comment by jayvdb** [comment]
wah; why not squashed?

**Comment by Mixih** [comment]
oh yes, good point.. they're all closely related

**Comment by Makman2** [comment]
@jayvdb shall I squash? ...

File: bears/python/PyFlakesBear.py [hunk 1]
```diff
@@ -6,8 +6,8 @@
 @linter(executable='pyflakes',
         use_stderr=True,
         output_format='regex',
-        output_regex=r'(?P<filename>.*):(?P<line>\d+):'
-                     r'[(?P<column>\d+):|?]*(?P<severity>)\s(?P<message>.*)\n',
```
Comment (jayvdb) [inline bears/python/PyFlakesBear.py]: this line doesnt need to be modified.
changing it makes me waste time looking for a obscure change to the regex, when there is none.

**Comment by jayvdb** [comment]
reack 3d7e50d

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**